query.sql

```diff
@@ -1,56 +1,58 @@
 # query #1
 # no inbex 32-93
 # index 32-36
-SELECT em.employee_id, em.first_name, em.last_name
-FROM employee em
-ORDER BY em.last_name;
+SELECT e.employee_id, e.first_name, e.last_name
+FROM employee as e
+ORDER BY e.employee_id;
 
 # query #2
 # no inbex 2,58-2,78
 # index 2,42-2,59
-SELECT em.employee_id, em.first_name, em.last_name, AVG(s.amount)
-FROM employee em
-INNER JOIN salary s on em.employee_id = s.employee_id
-GROUP BY em.employee_id;
+SELECT e.employee_id, e.first_name, e.last_name, AVG(s.amount)
+FROM employee as e
+INNER JOIN salary as s on e.employee_id = s.employee_id
+GROUP BY e.employee_id;
 
 # query #3
 # no inbex 3,32-3,111
 # index 2,994-2,996
 SELECT p.name, AVG(s.amount) as avarage, MAX(s.amount) as max
-FROM salary s
-INNER JOIN employee em on s.employee_id = em.employee_id
-INNER JOIN position p on em.position_id = p.position_id
-GROUP BY  p.name
-ORDER BY p.name;
+FROM salary as s
+INNER JOIN employee as e on s.employee_id = e.employee_id
+INNER JOIN position as p on e.position_id = p.position_id
+WHERE s.salary_date >= DATE_SUB(CURDATE(),INTERVAL 1 YEAR)
+GROUP BY p.position_id
+ORDER BY p.position_id;
 
 # query #4
 # no inbex 2,467-2,761
 # index 2,477-2,615
-SELECT t.number, SUM(td.income) as sum, AVG(td.income) as avarage, COUNT(td.transport_data_id) as count_day
+SELECT e.last_name, SUM(td.income) as sum, COUNT(td.employee_id) as count_day
 FROM transport_data td
-INNER JOIN transport t on td.transport_id = t.transport_id
-GROUP BY td.transport_id;
+INNER JOIN employee as e on td.employee_id = e.employee_id
+GROUP BY e.employee_id;
 
 # query #5
 # no inbex 2,747-2,892
 # index 2,761-2,765
-SELECT e.last_name, SUM(td.income) as sum, COUNT(td.employee_id) as count_day
-FROM transport_data td
-INNER JOIN employee e on td.employee_id = e.employee_id
-GROUP BY e.last_name;
+SELECT t.number, SUM(td.income) as sum, AVG(td.income) as avarage, COUNT(td.transport_data_id) as count_day
+FROM transport_data as td
+         INNER JOIN transport as t on td.transport_id = t.transport_id
+GROUP BY td.transport_id
+ORDER BY COUNT(td.transport_data_id), AVG(td.income), SUM(td.income) DESC;
```

Comment:
@pandaUbuntu 
1) Does this really work? Must use aliases in `ORDER BY`, but not to process the same things twice:
`ORDER BY count_day, avarage, sum`. 
2) Column aliases are not self-descriptive.
- sum > total_income
- avarage > average_income
- count_day > count_days, work_days, total_days_worked (up to you)